docs/pages/blog/turbo-1-2-0.mdx

```diff
@@ -0,0 +1,112 @@
+
+## Improved Internal Scheduler and Graph
+
+When using `turbo run`, every `package.json` task is added to an internal graph to map dependencies based on the inferred relationships defined in your Turborepo `pipeline`. This task graph allows Turborepo to efficiently schedule incremental concurrent task running and cache task outputs for later use.
+
+We have made major improvements to the internal task scheduler and resulting graph structure, resulting in better performance and a better developer experience. For example, in many cases, you will no longer need to use `--include-dependencies`. Instead, after specifying your task entry points, the new and improved graph will automatically handle this graph resolution for you.
+
+## Cache Outputs Integrity and Signature Verification
+
+You can now configure Turborepo to sign remote cache outputs using HMAC-SHA256 with a secret key before uploading them to the Remote Cache. When Turborepo downloads signed cache artifacts, it will now verify the artifact's integrity and authenticity. Any artifact that fails to verify will be ignored, discarded, and treated as a cache miss by Turborepo.
+
+To enable this feature, set the `remoteCache` options in your `turbo.json` config file to include `signature: true`. Then specify your secret key by declaring the `TURBO_REMOTE_CACHE_SIGNATURE_KEY` environment variable.
+
+```jsonc
+{
+  "$schema": "[https://turborepo.org/schema.json](https://turborepo.org/schema.json)",
+  "remoteCache": {
+    // Indicates if signature verifcation is enabled.
+    "signature": true
+  }
+}
+```
+
+## Other bug fixes and improvements
+
+- `--sso-team` flag now enables teams with SAML tokens to log in through `turbo login` with correct team permissions
+- `--log-output` flag allows you to control what logs are printed to the terminal, and when, allowing you to focus only on what's new
+- `FORCE_COLOR` environment variable is now supported
+- `TURBO_FORCE=true` environment variable will now force execution
+- `--remote-only` and `TURBO_REMOTE_ONLY=true` will tell `turbo` to only use Remote Caching
+- We now show `>>> FULL TURBO` when there's at least one task attempted
+
+## We're hiring!
+
+The Turborepo team at [Vercel](https://vercel.com) is hiring! We're up to 5 core team members already this year and are looking to hire even more. We're specifically looking for full time [Senior Build Systems Engineers](https://vercel.com/careers/senior-build-systems-engineer-turborepo-us-4383106004).
+
+## What's next?
+
+Along with seamless incremental adoption/migration and speeding up CI/CD, we've been focusing on improving Turborepo's day-to-day ergonomics, security, and observability. The new `--filter` flag, signed artifacts, and dry runs are important steps toward those goals.
+
+Next up, we'll be focusing an enhanced local development experience, codebase automations, and overall CLI performance.
```

Comment:
Do you want to give a shoutout to the contributors like the Next.js releases?

Reply:
Yes will aggregate them all